bee-message/tests/milestone.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_common::packable::Packable;
+use bee_message::{milestone::Milestone, prelude::*};
+use bee_test::rand::{milestone::rand_milestone, number::rand_number};
+
+use std::str::FromStr;
+
+const MESSAGE_ID: &str = "9e23e9fccb816af4ad355c27d904b6a6e88618e0bed1b640df3d4c19f4579bc9";
+
+#[test]
+fn debug_impl() {
+    let milestone = Milestone::new(MessageId::from_str(MESSAGE_ID).unwrap(), 0);
+
+    assert_eq!(
+        format!("{:?}", milestone),
+        "Milestone { message_id: MessageId(9e23e9fccb816af4ad355c27d904b6a6e88618e0bed1b640df3d4c19f4579bc9), timestamp: 0 }",
+    );
+}
+
+#[test]
+fn packed_len() {
+    let milestone = rand_milestone();
+    assert_eq!(milestone.packed_len(), milestone.pack_new().len());
```

Comment:
I would also add a check with the actual expected hardcoded length. We don't have the guarantee both packed_len and pack_new().len() aren't faulty the same way atm

Reply:
Yep, will do.